ui/client/package.json

```diff
@@ -81,41 +81,41 @@
     "uuid": "^3.3.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.8.0",
-    "@babel/core": "^7.8.0",
-    "@babel/plugin-proposal-class-properties": "^7.8.0",
-    "@babel/plugin-proposal-decorators": "^7.8.0",
-    "@babel/plugin-proposal-do-expressions": "^7.8.0",
-    "@babel/plugin-proposal-export-default-from": "^7.8.0",
-    "@babel/plugin-proposal-export-namespace-from": "^7.8.0",
-    "@babel/plugin-proposal-function-bind": "^7.8.0",
-    "@babel/plugin-proposal-function-sent": "^7.8.0",
-    "@babel/plugin-proposal-json-strings": "^7.8.0",
-    "@babel/plugin-proposal-logical-assignment-operators": "^7.8.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.8.0",
-    "@babel/plugin-proposal-numeric-separator": "^7.8.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.8.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.8.0",
-    "@babel/plugin-proposal-pipeline-operator": "^7.8.0",
-    "@babel/plugin-proposal-throw-expressions": "^7.8.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.0",
-    "@babel/plugin-syntax-import-meta": "^7.8.0",
-    "@babel/preset-env": "^7.8.2",
-    "@babel/preset-react": "^7.8.0",
-    "@babel/preset-typescript": "^7.8.0",
-    "@testing-library/react": "^9.4.1",
-    "@types/classnames": "^2.2.9",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.7",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
```

Comment:
Maybe we should extract common versions, like 7.8.3, 7.8.7 etc 7.8.3 to some variable?
Then when someone will update this version all librarier will aoutomatically be updated